JSDemos/Demos/Autocomplete/Overview/Angular/app/app.component.ts

```diff
@@ -49,16 +75,15 @@ export class AppComponent {
     this.names = service.getNames();
     this.surnames = service.getSurnames();
     this.positions = service.getPositions();
-    this.cities = service.getCities();
     this.position = this.positions[0];
   }
 
   updateEmployeeInfo() {
     let result = '';
     result += (`${this.firstName || ''} ${this.lastName || ''}`).trim();
-    result += (result && this.position) ? (`, ${this.position}`) : this.position;
-    result += (result && this.city) ? (`, ${this.city}`) : this.city;
-    result += (result && this.state) ? (`, ${this.state}`) : this.state;
+    result += (result && this.position) ? (`, ${this.position}`) : this.position || '';
```

Comment:
Did u want to handle the case when prop is not defined? If so, let's use nullish coalescing operator `??`
```suggestion
    result += (result && this.position) ? (`, ${this.position}`) : this.position ?? '';
```

Reply:
@markallenramirez  i'm sorry but seems like this changes are breaking. If our demo env does not support `hasOwn` it can also not support nullish coalescing. Let's revert it